soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -164,7 +167,7 @@ int GetRandomShopPrice() {
         max = 199; // 995/5
     }
     if (max != 0) {
-        return GetPriceFromMax(max);
+        return Settings::ShopsanityPricesAffordable.Is(true) ? GetPriceAffordable() : GetPriceFromMax(max);
```

Comment:
and then this line can just be 
```suggestion
        return GetPriceFromMax(max);
```

Reply:
Yeah, I caught that one.